src/components/RsvpBadge/RsvpBadge.js

```diff
@@ -6,16 +6,13 @@ const toK = (n) => {
 };
 
 const RsvpBadge = ({ totalAccepted, center, type }) => {
-  const alignStyle = center ? styles.wrapperCenter : styles.wrapperRight,
-    spanRSVP = <span className={styles.shortText}>{totalAccepted}+ Going</span>
+  const alignStyle = center ? styles.wrapperCenter : styles.wrapperRight;
+  const spanRSVP = <span className={styles.shortText}>{totalAccepted}+ Going</span>;
   if (type === 'mobileOnly') {
     return spanRSVP;
   }
   return (
     <div>
-      {
-        (type !== 'desktopOnly') && spanRSVP
-      }
```

Comment:
I didn't have a chance to really investigate exactly what the `desktopOnly` and `mobileOnly` values due, but I know that we were seeing the mobile/text-based version of the RSVP count show up the "tablet" (md, I think?) and mobile widths, and removing the 3 lines above fixed the issue without breaking anything else I could find. However, It would be good if @pdw207 could confirm that I'm not breaking something else in removing these lines.

Reply:
This added feature is great! do you think it might be best to make to match to the styling of the mobile RSVP tags (simple grey text) for both mobile and desktop views?

Reply:
I actually think that would be a pretty good idea, my vote is to keep them consistent. But I'd put that work into a separate ticket.

Reply:
Ahh, ok, looks like those lines are indeed necessary, just had to add a property to one of the RSVP tags to the EventCard component.